packages/domain-picker/src/hooks/use-domain-availabilities.ts

```diff
@@ -1,11 +1,8 @@
+import { DomainSuggestions } from '@automattic/data-stores';
 import { useSelect } from '@wordpress/data';
-import { DOMAIN_SUGGESTIONS_STORE } from '../constants';
-import type { DomainSuggestionsSelect } from '@automattic/data-stores';
```

Comment:
The definition of this type can now be removed completely.

Reply:
done!